great_expectations/data_context/data_context/abstract_data_context.py

```diff
@@ -1823,7 +1823,146 @@ def get_validator(  # noqa: PLR0913
                 include_rendered_content=include_rendered_content
             )
         )
+        expectation_suite = self._get_expectation_suite_from_inputs(
+            expectation_suite=expectation_suite,
+            expectation_suite_name=expectation_suite_name,
+            create_expectation_suite_with_name=create_expectation_suite_with_name,
+            expectation_suite_id=expectation_suite_id,
+            include_rendered_content=include_rendered_content,
+        )
+        batch_list = self._get_batch_list_from_inputs(
+            datasource_name=datasource_name,
+            data_connector_name=data_connector_name,
+            data_asset_name=data_asset_name,
+            batch=batch,
+            batch_list=batch_list,
+            batch_request=batch_request,
+            batch_request_list=batch_request_list,
+            batch_data=batch_data,
+            data_connector_query=data_connector_query,
+            batch_identifiers=batch_identifiers,
+            limit=limit,
+            index=index,
+            custom_filter_function=custom_filter_function,
+            sampling_method=sampling_method,
+            sampling_kwargs=sampling_kwargs,
+            partitioner_method=partitioner_method,
+            partitioner_kwargs=partitioner_kwargs,
+            runtime_parameters=runtime_parameters,
+            query=query,
+            path=path,
+            batch_filter_parameters=batch_filter_parameters,
+            batch_spec_passthrough=batch_spec_passthrough,
+            **kwargs,
+        )
+        return self.get_validator_using_batch_list(
+            expectation_suite=expectation_suite,  # type: ignore[arg-type]
```

Comment:
I wonder if we can remove this type ignore since we are passing an ExpectationSuite here.

Reply:
Done.